test/test_files/zalesak/zalesak.yaml

```diff
@@ -0,0 +1,13 @@
+
+exawind:
+  nalu_wind_inp:
+    - zalesak-nalu.yaml
+  amr_wind_inp: zalesak-amr.inp
+  num_timesteps: 60
```

Comment:
How fast does this thing run? 60 timesteps seems like a lot.  If it takes longer than 10 seconds I'd say dial this back for the repo.

Reply:
10.28 sec locally. The iterations don't take long because there are no pressure solves